### PR TITLE
Work on #1411.

### PR DESCRIPTION
**Pull request by mjordan**
**GitHub Issue**: https://github.com/Islandora/documentation/issues/1411

# What does this Pull Request do?

Adds an additional field for FITS data, the PUID field populated by Droid.

# What's new?

This PR parses out the value of the `<externalIdentifier toolname="Droid" toolversion="6.4" type="puid">fmt/12</externalIdentifier>` element of the FITS XML, and then adds a new "Droid" member to the array of metadata already extracted from the FITS XML.

# How should this be tested?

1. check out this PR
1. add a new node and accompanying media
1. look for the following at the end of the FITS metadata output:

Capture

(Value with vary depending on the format of the media.)

# Interested parties
@ajstanley, @Islandora/8-x-committers

### CODE REVIEW

File: src/Services/XMLTransform.php [hunk 1]
```diff
@@ -94,6 +100,7 @@ public function transformFits($input_xml) {
       ];
     }
     else {
+      $fits_metadata['Droid'] = ['PUID' => $puid];
```
Comment (ajstanley) [inline src/Services/XMLTransform.php]: If the $puid is wrapped in a conditional (line 88), should this also be wrapped?

Comment (mjordan) [reply src/Services/XMLTransform.php]: Makes sense. Done!

**Comment by mjordan** [comment]
@ajstanley any more work needed on this?

**Comment by ajstanley** [comment]
Merged!  Thanks @mjordan!

**Pull request merged**